### PR TITLE
Removed accents, muted

### LINKED ISSUE #26: Cut All Accents
### Maintainer Feature Request

**Feature description:** remove all automated accents except for those like pain stuttering. This includes species and trait accents.

**Lore justification (if applicable):** 

**Additional context:** They're better when hand-written and shouldn't be forced on characters as we can assume Galactic Common is relatively well suited for the mouths of every species involved. If you do want them, roleplay them out.

### LINKED ISSUE #27: Cut Muted until Sign Language Implemented (If Ever)
### Maintainer Feature Request

**Feature description:** remove the mute trait.

**Lore justification (if applicable):** NT wouldn't hire someone who can't communicate, sorry Komi.

**Additional context:** If sign language is ever implemented I would be fine with this being reimplemented.

### PR DESCRIPTION
**Pull request by nearlyNonexistent**
## About the PR

Accents gone from all clothing & traits selection.
Closes #26 and closes #27.

## Why / Balance

Lore issues & some are just stupid (ushanka).

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- remove: Removed all accents that don't make sense in our lore, like species accents, or ushanka accents.

### CODE REVIEW

File: Resources/Locale/en-US/traits/traits.ftl
Comment (LonelyHowls) [inline Resources/Locale/en-US/traits/traits.ftl:1]: You left an extra "-" in "trait-pirate-**-**name 

Also, did we wanna comment this out entirely?

File: Resources/Prototypes/Nyanotrasen/Species/Oni.yml
Comment (LonelyHowls) [inline Resources/Prototypes/Nyanotrasen/Species/Oni.yml:1]: Why is this in the accents PR?

File: Resources/Prototypes/Nyanotrasen/metempsychoticHumanoids.yml
Comment (LonelyHowls) [inline Resources/Prototypes/Nyanotrasen/metempsychoticHumanoids.yml:1]: Why is this in the accents PR?

File: Resources/Prototypes/Species/arachnid.yml
Comment (LonelyHowls) [inline Resources/Prototypes/Species/arachnid.yml:1]: Why is this in the accents PR?

File: Resources/Locale/en-US/traits/traits.ftl
Comment (nearlyNonexistent) [reply Resources/Locale/en-US/traits/traits.ftl:1]: Oh, we shouldn't be changing the .ftl file, that's just for language translation. I'll remove the changes to this file entirely.

**Review by LonelyHowls — APPROVED** [review]
You get the Big Puppy Stamp of Pupproval.

**Pull request merged**